src/libs/ajax.js

```diff
@@ -214,7 +214,7 @@ const User = signal => ({
   },
 
   uploadAttachment: async file => {
-    const res = await fetch(`https://broadinstitute.zendesk.com/api/v2/uploads?filename=${file.name}`, {
+    const res = await fetchOk(`https://broadinstitute.zendesk.com/api/v2/uploads?filename=${file.name}`, {
```

Comment:
Why did you change this to `fetchOk`? The whitelist service doesn't seem related to zendesk attachments.